clang/include/clang/AST/ASTDumper.h

```diff
@@ -45,12 +49,16 @@ class ASTDumper : public ASTNodeTraverser<ASTDumper, TextNodeDumper> {
   void VisitVarTemplateDecl(const VarTemplateDecl *D);
 
   // Checked C specific methods.
+  void VisitForWhereClause(const Stmt *Stmt);
```

Comment:
Can the function name be changed to `VisitWhereClauseOnStmt`?

Reply:
Sure! I will follow this.